CMakeLists.txt

```diff
@@ -12,12 +12,16 @@ option(THREADSAFE "Threadsafe" ON)
 option(USE_SSH "Use SSH" OFF)
 option(USE_HTTPS "Use HTTPS" OFF)
 option(USE_THREADS "Use threads" ON)
-if (APPLE)
-    set(OPENSSL_ROOT_DIR "/opt/homebrew/opt/openssl@1.1/")
-endif()
-if (UNIX)
-    set(OPENSSL_ROOT_DIR "/usr/local/ssl/")
-endif()
+
+if (NOT DEFINED OPENSSL_ROOT_DIR)
```

Comment:
Unrelated, but it was annoying we were essentially ignoring any custom value of OPENSSL_ROOT_DIR that the caller might have preferred to use.